assets/css/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.navegation-menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  width: 100%;
+}
+
+.humburguer {
+  padding-right: 20px;
+  margin-top: 20px;
+  color: black;
+}
+
+.headline-section {
+  background-image: url(../img/header-illsutration-mobile.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 712px;
+  padding-top: 92px;
+}
```

Comment:
- [ ] Kindly make sure to stick to the `Figma` design template as much as possible. Please include all the background images to much that of the `Figma` design template as it's important to help make your website look good and create a good user experience. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="380" alt="Screenshot 2023-06-02 at 10 43 30" src="https://github.com/Graciano1997/portofolio-graciano/assets/107192076/8be4d05d-eacc-439c-8358-e4c6235132dd">
</td>
<td>
<img width="308" alt="Screenshot 2023-06-02 at 10 52 33" src="https://github.com/Graciano1997/portofolio-graciano/assets/107192076/a462d363-49b3-4ed9-9d82-4f13fb862028">
</td>
</tr>
</table>

Reply:
Hello I should like to know in which size did you in your browser?

Reply:
@Graciano1997 Sorry for the late reply, mobile screen size is `375px`

Reply:
I come to thank you again because it improved my project 💯 💯  🙏 🙏